modern/src/MainPage.js

```diff
@@ -36,35 +36,96 @@ const useStyles = makeStyles(theme => ({
     },
     overflow: 'hidden',
   },
-  mapContainer: {
-    flexGrow: 1,
+  searchBox: {
+    position: 'absolute',
+    left: theme.spacing(1.5),
+    top: theme.spacing(10.5),
+    width: theme.dimensions.drawerWidthDesktop,
+    borderRadius: '0px',
+    [theme.breakpoints.down("md")]: {
+      top: theme.spacing(7),
+      left: '0px',
+      width: '100%'
+    }  
+  },
+  searchToolbar: {
+    paddingLeft: theme.spacing(2),
+    paddingRight: theme.spacing(2),
   },
+  deviceList: {
+    position: 'absolute',
+    left: theme.spacing(1.5),
+    top: theme.spacing(20),
+    width: theme.dimensions.drawerWidthDesktop,
+    borderRadius: '0px',
+    height: '100%',
+    maxHeight: `calc(100vh - ${theme.spacing(23)}px)`,
```

Comment:
What is the reason for this calculation? Can we not have it take the rest of the available space?

Reply:
maxHeight is defined because we have to show the menu at the bottom.

Reply:
Menu has a fixed height, so it should still work.